hedera-node/src/main/java/com/hedera/services/store/tokens/HederaTokenStore.java

```diff
@@ -584,14 +585,26 @@ public ResponseCodeEnum update(final TokenUpdateTransactionBody changes, final l
 
 	private AliasLookup checkNewAutoRenewAccount(final TokenUpdateTransactionBody changes) {
 		if (changes.hasAutoRenewAccount()) {
-			return lookUpAccountId(changes.getAutoRenewAccount(), INVALID_AUTORENEW_ACCOUNT);
+			final var autoRenewLookUp =  lookUpAccountId(changes.getAutoRenewAccount(), INVALID_ALIAS_KEY);
```

Comment:
should return `INVALID_AUTORENEW_ACCOUNT`

Reply:
I made it into 2 parts. first we look up the alias key. If found and its not a usable accountId we return `INVALID_AUTORENEW_ACCOUNT` but if that alias is not found we return `INVALID_ALIAS_KEY`. I open to change it though.